db/models.py

```diff
@@ -14,3 +14,34 @@ class Actor(models.Model):
 
     def __str__(self):
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
I think it is more useful to call "related_name" as "movies" than "actors" or "genres", then it will be more useful to understand! But I approve!

Reply:
I agree with Nazar. To get movies now in which actor is involved you use `actor.actors.all()`, change it to `actor.movies.all()`, the same with genres.